src/poetry/utils/authenticator.py

```diff
@@ -35,11 +35,7 @@ def __init__(self, config: "Config", io: Optional["IO"] = None) -> None:
 
     def _log(self, message: str, level: str = "debug") -> None:
         if self._io is not None:
-            self._io.write_line(
-                "<{level:s}>{message:s}</{level:s}>".format(
-                    message=message, level=level
-                )
-            )
+            self._io.write_line(f"<{level:s}>{message:s}</{level:s}>")
```

Comment:
I think we can ditch the `:s` here safely as the type on the method makes it clear those are already strings.